example-workflows/full-example.md

```diff
@@ -0,0 +1,96 @@
+ # Full example
```

Comment:
I think the name is confusing,  because it doesn't mean "full" as in "all options" but "including tutorial".

It's pretty useful, but I maybe it would be off best in the documentation: https://dokku.com/docs/deployment/continuous-integration/github-actions that can be linked